### PR TITLE
Fix internal links and redirects for two-factor move.

### PR DESCRIPTION
**Pull request by robotdan**
Related commit:
- https://github.com/FusionAuth/fusionauth-site/commit/2a8515229ce24f21567c7331ddf9f69d37d69c81

@mooreds and @lyleschemmerling can you check my changes to `redirects.json` and `docs-url-map.json` - not 100% sure on how those are utilized.

### CODE REVIEW

File: astro/src/content/blog/securely-implement-oauth-in-react.mdx [hunk 1]
```diff
@@ -30,7 +30,7 @@ In addition to React, we'll use a NodeJS backend. One major benefit of using a b
 
 <Aside type="tip" nodark="true">This blog post has been [superseded by a newer React/OAuth tutorial](/blog/2021/11/11/how-to-authenticate-your-react-app).</Aside>
 
-If you're in-the-know on OAuth, you're probably aware that some people use PKCE or the rightfully deprecated Implicit Flow to get around the Client Secret constraint. We're sticking to Authorization Code Flow, because setting up a server is the most secure solution for the majority of cases and quite simple once you know how. Don't know what any of that means? Don't worry about it — you don't need to know anything about OAuth to follow this example.
```
Comment (lyleschemmerling) [inline astro/src/content/blog/securely-implement-oauth-in-react.mdx:33]: Are all of these the result of two conflicting editor settings? Someone with trailing spaces by default and yours removes them? If so we should figure out how to get that consistent.

**Review by lyleschemmerling — APPROVED** [review]

File: astro/docs-url-map.json [hunk 2]
```diff
@@ -27,7 +27,7 @@
   "/docs/v1/tech/apis/identity-providers/twitter": "/docs/apis/identity-providers/twitter",
   "/docs/v1/tech/apis/identity-providers/xbox": "/docs/apis/identity-providers/xbox",
   "/docs/v1/tech/apis/messengers/generic": "/docs/apis/messengers/generic",
-  "/docs/v1/tech/apis/two-factor": "/docs/apis/messengers/two-factor",
+  "/docs/v1/tech/apis/two-factor": "/docs/apis/two-factor",
```
Comment (mooreds) [inline astro/docs-url-map.json:30]: You don't have a redirect for `/docs/apis/messengers/two-factor`=> `/docs/apis/two-factor` which is one of my main concerns. So I'd want to have that added back in.

File: astro/src/content/blog/securely-implement-oauth-in-react.mdx (hunk 1 above)
Comment (mooreds) [reply astro/src/content/blog/securely-implement-oauth-in-react.mdx:33]: I don't use intellij, I use vim and sometimes have whitespace on a line.

Other folks use other editors.

Is this something we can automate with a GH script?

Agree that this doesn't seem to be good to wrap into this PR, but not sure that's a blocker.

**Review by mooreds — CHANGES_REQUESTED** [review]

File: astro/src/content/blog/securely-implement-oauth-in-react.mdx (hunk 1 above)
Comment (robotdan) [reply astro/src/content/blog/securely-implement-oauth-in-react.mdx:33]: Probably - IJ is configured to remove trailing white space. I would prefer that be the default on any editor we use to edit docs.

File: astro/docs-url-map.json (hunk 2 above)
Comment (robotdan) [reply astro/docs-url-map.json:30]: What is the difference between `docs-url-map.json` and `redirects.json`?

In `redirects.json` I do have this accounted for, so are these redundant and I need to add redirects to both, or is one just a map used for something else?

`redirects.json`

```
"/docs/v1/tech/apis/two-factor": "/docs/apis/two-factor",
"/docs/apis/messengers/two-factor": "/docs/apis/two-factor",
```

Comment (mooreds) [reply astro/docs-url-map.json:30]: My apologies, I was looking at the wrong file. Your redirects look great.

I don't have any idea what docs-url-map.json is.

Comment (mooreds) [reply astro/docs-url-map.json:30]: Resolving.

**Review by mooreds — APPROVED** [review]

**Pull request merged**